lambdas/service/app.py

```diff
@@ -1636,9 +1636,7 @@ def generate_manifest(event, context):
                 replica — for those that do — will be used.
 
                 All query parameters not mentioned above are forwarded to the
```

Comment:
I don't think this sentence is actually true anymore, or only true for DSS. Could you check, please?

Reply:
It's no longer true, since `validate_params()` here doesn't allow extra parameters.

I'll remove this line in a separate commit.